### ISSUE TITLE
game 'packages' generator and consumer

### ISSUE THREAD

**Issue by loklaan** [demscode/bearded-lama#17]
These packages consist of a stringy object, like JSON or XML. They will be stored in the db as opposed to the game files. Ideally they can be easily traversed for URLs.

_Example **package.json**_

``` json
{
  "main": "/public/:gameid/js/game.js",
  "js": [
    "/public/:gameid/js/state.js",
    "/public/:gameid/js/entity.js",
    "/public/:gameid/js/levels.js",
    "/public/:gameid/js/ai.js"
    ],
  "images": [
    "/public/:gameid/assets/background.png",
    "/public/:gameid/assets/sprites.png",
    "/public/:gameid/assets/loading.png"
    ],
  "sounds": [
    "/public/:gameid/assets/loop.ogg",
    "/public/:gameid/assets/shot.ogg",
    "/public/:gameid/assets/dead.ogg"
    ]
}
```

**Comment by loklaan** [comment]
Most html5 devs who partake in game james usually use git (and github) for source control. I wonder how hard it would be to offer an option for the user account to be linked with github account, to then 'attach' their repos to a game submission...

Person:
1. makes account on BL
2. links account to [GitHub account](https://help.github.com/articles/connecting-with-third-party-applications)
3. opens new game submission or edit existing submission
4. instead of submitting an archive, person 'attaches' github repo
- _pushing to repo can active a [webhook](https://help.github.com/articles/creating-webhooks) for BL to cache latest_
- _person still follows game package rules_
- _coding wouldn't be too hard with [existing lib](https://github.com/octokit/octokit.net)_

**Comment by loklaan** [comment]
Another field that specifies the html element id or class that the games main javascript calls into should be added.

Else, if some other way of inserting the game into an element is used that forgoes an explicit id or class perhaps support that or make stricter specification to match the asserted expectation mention above.

**Comment by loklaan** [comment]
In-app documentation available from 91a0a16
